cyclonedx/model/component.py

```diff
@@ -204,44 +866,59 @@ class ComponentEvidence:
 
     def __init__(
         self, *,
+        identity: Optional[Iterable[Identity]] = None,
+        occurrences: Optional[Iterable[Occurrence]] = None,
+        callstack: Optional[CallStack] = None,
         licenses: Optional[Iterable[License]] = None,
         copyright: Optional[Iterable[Copyright]] = None,
     ) -> None:
+        self.identity = identity or []  # type:ignore[assignment]
+        self.occurrences = occurrences or []  # type:ignore[assignment]
+        self.callstack = callstack
         self.licenses = licenses or []  # type:ignore[assignment]
         self.copyright = copyright or []  # type:ignore[assignment]
 
-    # @property
-    # ...
-    # @serializable.view(SchemaVersion1Dot5)
-    # @serializable.xml_sequence(1)
-    # def identity(self) -> ...:
-    #    ... # TODO since CDX1.5
-    #
-    # @identity.setter
-    # def identity(self, ...) -> None:
-    #    ... # TODO since CDX1.5
+    @property
+    @serializable.view(SchemaVersion1Dot6)
```

Comment:
where is the support for CDX 1.5? 

```suggestion
    @serializable.view(SchemaVersion1Dot6)
    @serializable.view(SchemaVersion1Dot5)
```

Reply:
Okay, By the way my change set is ready! for SchemaVersion1Dot5

Reply:
pushed fixes and refactor based on my findings during the review.
for the EvidenceIdentiry - the 1.5/1.6 edge case is still open.
see https://github.com/CycloneDX/cyclonedx-python-lib/pull/810/files#diff-033c9438d226c6869174d505bcba9922051bab9f1844f9682b3a2b9aaf52d0f2R654-R655